webapp/src/main/java/com/box/l10n/mojito/service/assetExtraction/AssetMappingService.java

```diff
@@ -147,6 +149,7 @@ public void mapAssetTextUnitAndCreateTMTextUnit(Long assetExtractionId, Long tmI
                     unmappedAssetTextUnit.getPluralForm(),
                     unmappedAssetTextUnit.getPluralFormOther());
 
+            addTMTextUnit.setCreatedByUser(user);
```

Comment:
I'd pass the user down to `addTmTextUnit()` and if it is `null` will use `auditorAwareImpl.getCurrentAuditor()`

Reply:
move createdByUser to addTMTextUnit